examples/tensorflow-privacy/tf_privacy/client_app.py

```diff
@@ -103,48 +93,34 @@ def evaluate(self, parameters, config):
         return loss, len(self.x_test), {"accuracy": accuracy}
 
 
-def client_fn_parameterized(
-    partition_id,
-    noise_multiplier,
-    l2_norm_clip=1.0,
-    num_microbatches=64,
-    learning_rate=0.01,
-    batch_size=64,
-):
-    def client_fn(cid: str):
-        model = tf.keras.Sequential(
-            [
-                tf.keras.layers.InputLayer(input_shape=(28, 28, 1)),
-                tf.keras.layers.Conv2D(32, kernel_size=(3, 3), activation="relu"),
-                tf.keras.layers.MaxPooling2D(pool_size=(2, 2)),
-                tf.keras.layers.Conv2D(64, kernel_size=(3, 3), activation="relu"),
-                tf.keras.layers.MaxPooling2D(pool_size=(2, 2)),
-                tf.keras.layers.Flatten(),
-                tf.keras.layers.Dense(128, activation="relu"),
-                tf.keras.layers.Dense(10, activation="softmax"),
-            ]
-        )
-        train_data, test_data = load_data(
-            partition_id=partition_id, batch_size=batch_size
-        )
-        return FlowerClient(
-            model,
-            train_data,
-            test_data,
-            noise_multiplier,
-            l2_norm_clip,
-            num_microbatches,
-            learning_rate,
-            batch_size,
-        ).to_client()
-
-    return client_fn
-
-
-appA = ClientApp(
-    client_fn=client_fn_parameterized(partition_id=0, noise_multiplier=1.0),
-)
+def client_fn(context: Context):
+    model = Net()
+    model.build(input_shape=(None, 28, 28, 1))
 
-appB = ClientApp(
-    client_fn=client_fn_parameterized(partition_id=1, noise_multiplier=1.5),
-)
+    partition_id = context.node_config["partition-id"]
+
+    l2_norm_clip = 1.0
+    num_microbatches = 64
+    learning_rate = 0.01
+    batch_size = 64
```

Comment:
one idea would be to add this to the `pyproject.toml`. Then, it can also be passed to `FlowerClient` below like this:

```
return FlowerClient(..., config=context.run_config, ...`)
```

and avoid have "duplicated" lines (currently we have almost the same thing 4 times)